### ISSUE TITLE
Not able to navigate from Source Trail IDE into Qt Creator 4.10.1 

### ISSUE THREAD

**Issue by ankitthakar** [CoatiSoftware/qtc-sourcetrail#17]
I was just trying to use Source trail in Qt 4.10.

And I was not able to navigate from Source Trail to Qt creator.
I have also tried plugins based on this link [https://github.com/CoatiSoftware/qtc-sourcetrail]
But it didn't worked...

**Comment by mlangkabel** [comment]
Does it work the other way round? I mean, can you right-click your code in Qt creator and send the location to Sourcetrail?

You can try to configure different ports on both sides. Maybe the default ports are used on your machine.

**Comment by ankitthakar** [comment]
No, it doesn't work other way round also. 
Can you please tell me how to change the ports?

**Comment by mlangkabel** [comment]
In Qt Creator: In the menu Tools -> Options... -> Sourcetrail
In Sourcetrail: in the menu Edit -> Preferences... -> Plugin

**Comment by nyanpasu64** [comment]
I cannot navigate from Sourcetrail to Qt Creator either.

In Menu bar, Tools→Sourcetrail, "Send Location to Sourctrail" works, but "Server Status: Stopped". If I try to start the server, I get message `Sourcetrail Plugin TCP Server - Could not listen to port 6666`.

Port 6666 is not occupied. `python -m http.server 6666` binds fine.

EDIT: it's occupied?

```
nyanpasu@win /mnt/c/Users/nyanpasu/src/qt-everywhere-src-5.14.2> nc -zv localhost 6666
Connection to localhost 6666 port [tcp/*] succeeded!
```

EDIT2: Sublime Text's Sourcetrail plugin was occupying port 6666.

**Closing Sublime Text or disabling the Sourcetrail plugin fixed Qt Creator navigation.**

----

I'm on Windows 10 x64, Qt Creator 4.12.1 (32-bit), and Sourcetrail plugin 0.8.3.

**Comment by mlangkabel** [comment]
@nyanpasu64: Do I understand that correctly? You had both, QtCreator and Subline running (including their Sourcetrail plugins) and when sending a location from Sourcetrail, it went to Sublime and that caused QtCreator NOT to receive the command?

**Comment by nyanpasu64** [comment]
Right. Sublime Text and Qt Creator both had their plugins installed, but Sublime Text was occupying port 6666 and preventing Qt Creator from using that port. And Sourcetrail said it was connected to Sublime only. I'm not sure if jumping from Sourcetrail to Sublime worked (IIRC it didn't work, but I'm not using Sublime for this project).
